src/Console/Command/ObfuscateData.php

```diff
@@ -0,0 +1,305 @@
+<?php namespace Ushahidi\Console\Command;
+
+/**
+ * Ushahidi Obfuscate Data console command
+ *  - This overwrites specific data within the current database
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Console
+ * @copyright  2018 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+use Illuminate\Console\Command;
+use Ushahidi\Core\Entity\PostRepository;
+use Ushahidi\Core\Entity\ContactRepository;
+use Ushahidi\Core\Entity\ConfigRepository;
+use Ushahidi\Core\Entity\UserRepository;
+use Ushahidi\App\Multisite;
+use Ohanzee\DB;
+use Faker;
+use Database;
+
+class ObfuscateData extends Command
+{
+    private $postRepository;
+    private $contactRepository;
+
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'db:obfuscatedata';
+
+    /**
+     * The console command signature.
+     *
+     * @var string
+     */
+    protected $signature = 'db:obfuscatedata';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Obfuscates selected PII and removes various config keys from database';
+
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    public function handle()
+    {
+        //set up repos
```

Comment:
Not significant, but even though we can't do constructor injection, I'd generally put the `service()` calls in __construct. Just to separate them from the actual handler code

Reply:
updated